### PR TITLE
🧪 [Add missing error path test for negative length in generateShuffledIndices]

### PR DESCRIPTION
**Pull request by is0692vs**
🎯 **What:** The testing gap addressed was that `generateShuffledIndices` safely handled `length <= 0`, but the tests only covered `length === 0`. A test case handling negative lengths (`-5`) was missing.

📊 **Coverage:** Added a test case `handles negative length` which passes `-5` as the length. This guarantees coverage for scenarios where length drops below zero.

✨ **Result:** Test coverage for this pure utility function is improved by verifying all bounds `(length < 0, length === 0, length > 0)`, creating a stronger safety net against future regressions.

---
*PR created automatically by Jules for task [2383055690009912666](https://jules.google.com/task/2383055690009912666) started by @is0692vs*

<details open><summary><h3>Greptile Summary</h3></summary>

このPRは `generateShuffledIndices` 関数に対して、負の長さ（`-5`）を渡した場合に空配列を返すことを検証するテストケースを追加します。テスト自体は正確ですが、`pr_description.txt` がリポジトリにコミットされており、これはソースコードではないため削除が必要です。

- `pr_description.txt` がリポジトリのルートに追加されており、不要なファイルです。マージ前に削除してください。
</details>

<details open><summary><h3>Confidence Score: 4/5</h3></summary>

- `pr_description.txt` の不要なファイルを削除すれば安全にマージ可能です。
- テスト追加自体は正確で問題ありませんが、`pr_description.txt` というリポジトリに含めるべきでないファイルがコミットされているため、マージ前に対応が必要です。
- `pr_description.txt` — リポジトリに含めるべきでないファイルであり、削除が必要です。
</details>

<details open><summary><h3>Important Files Changed</h3></summary>

| Filename | Overview |
|----------|----------|
| packages/web-app-vercel/contexts/__tests__/PlaylistPlaybackContext.test.tsx | 負の長さ（-5）を渡した場合に空配列を返すことを検証するテストケースを追加。実装上 `Array.from({ length: -5 })` は `[]` を返すため、テストの期待値は正確。 |
| pr_description.txt | PR説明テキストがリポジトリにコミットされており、ソースコードではない不要なファイル。マージ前に削除が必要。 |

</details>

</details>

<h3>Flowchart</h3>

```mermaid
%%{init: {'theme': 'neutral'}}%%
flowchart TD
    A["generateShuffledIndices(length, currentIndex?)"] --> B{length <= 0?}
    B -- "はい (length=0 または length=-5)" --> C["return []"]
    B -- "いいえ (length>0)" --> D["Array.from でインデックス配列生成"]
    D --> E["Fisher-Yates シャッフル"]
    E --> F{currentIndex が有効範囲内?}
    F -- "はい" --> G["currentIndex を先頭に移動"]
    F -- "いいえ" --> H["そのまま返す"]
    G --> H["shuffled indices を返す"]

    style C fill:#90EE90
    style B fill:#FFD700
```

<details><summary>Prompt To Fix All With AI</summary>

`````markdown
This is a comment left during a code review.
Path: pr_description.txt
Line: 1-5

Comment:
**PR説明ファイルがリポジトリにコミットされている**

`pr_description.txt` はPRの説明文であり、ソースコードではないためリポジトリに含めるべきではありません。このファイルはGitHub/GitLabのPR本文フィールドに記載するべき内容であり、このままマージするとリポジトリのルートに不要なファイルが残ります。マージ前にこのファイルを削除してください。

How can I resolve this? If you propose a fix, please make it concise.
`````

</details>

<sub>Reviews (1): Last reviewed commit: ["🧪 \[testing improvement\] Add missing err..."](https://github.com/hiroki-org/audicle/commit/55182f8d37698908c0e0c4273fb62f22083cdce0) | [Re-trigger Greptile](https://app.greptile.com/api/retrigger?id=28308595)</sub>

> Greptile also left **1 inline comment** on this PR.

### CODE REVIEW

File: pr_description.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+🎯 **What:** The `generateShuffledIndices` function in `packages/web-app-vercel/contexts/PlaylistPlaybackContext.tsx` handles negative lengths properly by returning an empty array. However, this edge case was not tested in `packages/web-app-vercel/contexts/__tests__/PlaylistPlaybackContext.test.tsx`. A new test case for negative lengths has been added.
+
+📊 **Coverage:** The test coverage now includes the negative length scenario (e.g., length of `-5`) alongside empty array (length `0`) and positive values, ensuring that the early return statement functions reliably for any `length <= 0`.
+
+✨ **Result:** Increased testing confidence in the `generateShuffledIndices` pure function without changing its internal implementation.
```
[the remark this answers is not in the document]
Comment (is0692vs) [reply pr_description.txt]: レビュー解決ループ対応です。現時点（head 55182f8）ではこのPRでの追加対応は見送りとし、理由を明示したうえでスレッドを解決します。必要であれば再オープンをお願いします。

[the remark this answers is not in the document]
Comment (is0692vs) [reply pr_description.txt]: レビュー解決ループ対応です。現時点（head 55182f8）ではこのPRでの追加対応は見送りとし、理由を明示したうえでスレッドを解決します。必要であれば再オープンをお願いします。

[the remark this answers is not in the document]
Comment (is0692vs) [reply pr_description.txt]: 対応しました。commit 404f813 で pr_description.txt を削除し、PR説明用の一時ファイルをリポジトリから除外しました。併せて同種の補助ファイル reply_script.js も削除しています。

[the remark this answers is not in the document]
Comment (is0692vs) [reply pr_description.txt]: 対応しました。commit 404f813 で pr_description.txt を削除し、PR説明用の一時ファイルをリポジトリから除外しました。併せて同種の補助ファイル reply_script.js も削除しています。

[the remark this answers is not in the document]
Comment (is0692vs) [reply pr_description.txt]: 最終確認です。このPRは commit (404f813) で対応を反映済みです。必要な修正は反映し、スコープ外の指摘はthread内で理由を明示しています。

[the remark this answers is not in the document]
Comment (is0692vs) [reply pr_description.txt]: 最終確認です。このPRは commit (404f813) で対応を反映済みです。必要な修正は反映し、スコープ外の指摘はthread内で理由を明示しています。
